### PR TITLE
Updated proto files used by the PowerShell worker.

### PR DESCRIPTION
**Pull request by Francisco-Gamino**
Updated proto files used by the PowerShell worker.
* Updated subtree from https://github.com/azure/azure-functions-language-worker-protobuf. Branch: dev . Commit:2e9cf4b.
* Updated project to use Grpc.Core version 1.19.0

### CODE REVIEW

**Comment by Francisco-Gamino** [comment]
cc: @kashimiz @pragnagopa @SatishRanjan

File: protobuf/src/proto/shared/NullableString.proto [hunk 1]
```diff
@@ -0,0 +1,8 @@
+syntax = "proto3";
+// protobuf vscode extension: https://marketplace.visualstudio.com/items?itemName=zxh404.vscode-proto3
+
+message NullableString {
+	oneof string {
```
Comment (TylerLeonhardt) [inline protobuf/src/proto/shared/NullableString.proto]: Not sure if you want feedback on the proto files but the `oneof` block needs an extra space for it to be 2 spaces indented.

File: tools/helper.psm1 [hunk 2]
```diff
@@ -90,21 +90,42 @@ function New-gRPCAutoGenCode
...
             --csharp_out $outputDir `
             --grpc_out=$outputDir `
             --plugin=protoc-gen-grpc=$Script:grpc_csharp_plugin_Path `
-            --proto_path=$Script:identity_dir_Path `
+            --proto_path=$Script:protobuf_dir_Path `
             --proto_path=$Script:google_protobuf_tools_Path
+        }
+
+        # Generate other types
+        foreach ($folder in @(Get-ChildItem $Script:protobuf_dir_Path\* | Where-Object{$_.PsIsContainer -and $_.FullName -notmatch 'shared'}))
```
Comment (TylerLeonhardt) [inline tools/helper.psm1]:
```suggestion
        foreach ($folder in @(Get-ChildItem $Script:protobuf_dir_Path\* -Exclude *shared* -Directory))
```

Does this work?

File: protobuf/src/proto/shared/NullableString.proto (hunk 1 above)
Comment (Francisco-Gamino) [reply protobuf/src/proto/shared/NullableString.proto]: These files were pull from https://github.com/azure/azure-functions-language-worker-protobuf as is, so I will not update them there. These change will have to be made at that repo.

File: tools/helper.psm1 (hunk 2 above)
Comment (Francisco-Gamino) [reply tools/helper.psm1]: Yes, we could also do that. I will update the code.

**Comment by daxian-dbw** [comment]
#165 was submitted to resolve the build issue.

**Pull request merged**